SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,70 @@
+Supported Operations
+####################
+
+.. table::
+
+    +------------------+-----------------------------------------+--------------+-----------------------+
+    | **Visibility**   | **Operation**                           | **Protocol** | **Number of Parties** |
+    +==================+=========================================+==============+=======================+
+    | Public & Private | Addition                                | - Falcon     | - 3                   |
+    |                  |                                         | - FSS        | - 2                   |
+    |                  |                                         | - SPDZ       | - 2+                  |
```

Comment:
SPDZ is a protocol only for multiplication, matrix multiplication, conv2d

Reply:
Done, removed from addition and multiplication.